cobra/cmd/root.go

```diff
@@ -24,13 +26,20 @@ import (
 var (
 	// Used for flags.
 	cfgFile, userLicense string
+	fab                  bool
 
 	rootCmd = &cobra.Command{
 		Use:   "cobra",
 		Short: "A generator for Cobra based Applications",
 		Long: `Cobra is a CLI library for Go that empowers applications.
 This application is a tool to generate the needed files
 to quickly create a Cobra application.`,
+		Run: func(command *cobra.Command, args []string) {
+			if fab {
+				command.SetOutput(&lol.Writer{Output: os.Stdout, ColorMode: lol.ColorModeTrueColor})
```

Comment:
This defaults to `truecolor` which render well in *every* terminal.. the safest bet is to go with 256, but the coloring in that case isn't quite as smooth/rainbowy

Safe solution: `256`
Less safe but potentially prettier solution: `truecolor`

Thoughts?